tool/loadgen/main.go

```diff
@@ -18,6 +18,14 @@ const target = "http://app:8080"
 
 var hasRelay = os.Getenv("HAS_RELAY") == "true"
 
+type LoadGenOptions struct {
+	Path          string    `json:"path"`
+	ContainerName string    `json:"container_name"`
+	WaitGen       Generator `json:"wait_generator"`
```

Comment:
Perhaps the "wait" phase can be ignored from the parametrization? It does affect what Relay sees, and eventually what goes into the result, but if we wanted we could somehow exclude it from results.

The thing is this steps is only there to ensure the server is up and ready to accept requests.